### DISCUSSION TITLE
Importing fails Google UA: unable to authenticate your  Google Analytics account.

### DISCUSSION THREAD

**Discussion by dnlbln** [plausible/analytics#4170] [Self-Hosted Support] [answered]
Hi,

I installed a working version of the community edition on a server and followed the entire tutorials of setting up the OAuth and at the end when I try to import my data from google Universal analytics account I get the following error.

<img width="403" alt="Screenshot 2024-06-02 at 13 08 30" src="https://github.com/plausible/analytics/assets/3605743/081071f5-4bf1-4d8f-922a-6cb7214b0945">

Is there any way to debug this on the CE edition to figure out what is wrong? I tried importing it into the cloud edition and it worked totally fine.

I would truly appreciate if there was a way to solve this because at the end of the month all my analytics will be all deleted.

Thanks

**Comment by ruslandoga** [comment, marked as answer]
👋 @dnlbln 

You probably need to enable [Google Analytics Admin API](https://console.cloud.google.com/apis/api/analyticsadmin.googleapis.com) and [Google Analytics Data API](https://console.cloud.google.com/marketplace/product/google/analyticsdata.googleapis.com)

**Reply by ruslandoga** [reply]
This update to the integration guide might be useful too: https://github.com/plausible/community-edition/commit/6dc3dfac65dde92c57afffcb7ef45e50e9eaf4c6

**Reply by dnlbln** [reply]
This was the missing link. Enabling these 2 APIs has solved the issue. Thank you very much for the help!
